src/platform/user/profile/vap-svc/actions/transactions.js

```diff
@@ -132,11 +138,28 @@ export function refreshTransaction(
             transactionRefreshed?.data?.attributes?.metadata?.[0] ?? {};
           const errorCode = errorMetadata.code ?? 'unknown-code';
           const errorKey = errorMetadata.key ?? 'unknown-key';
+
+          const addressNames = Object.values(ANALYTICS_ADDRESS_FIELD_NAMES);
+          const phoneNames = Object.values(ANALYTICS_PHONE_FIELD_NAMES);
+          let sectionName = analyticsSectionName;
```

Comment:
setting this to analyticsSectionName to ensure it's never undefined since ANALYTICS_FIELD_MAP also includes

```
  INIT_VAP_SERVICE_ID: 'initialize-vet360-id',
  smsOptin: 'sms-optin',
  smsOptout: 'sms-optout',
```